src/rpc/rpcevo.cpp

```diff
@@ -279,14 +345,21 @@ void protx_register_prepare_help()
             "key and then passed to \"protx register_submit\". The prepared transaction will also contain inputs\n"
             "and outputs to cover fees.\n"
             "\nArguments:\n"
-            "1., 2., 3., ...         See help text of \"protx register\".\n"
-            "feeSourceAddress        See help text for \"fundAddress\" of \"protx register_fund\"\n"
+            + GetHelpString(1, "collateralHash")
+            + GetHelpString(2, "collateralIndex")
+            + GetHelpString(3, "ipAndPort")
+            + GetHelpString(4, "ownerAddress")
+            + GetHelpString(5, "operatorPubKey")
+            + GetHelpString(6, "votingAddress")
+            + GetHelpString(7, "operatorReward")
+            + GetHelpString(8, "payoutAddress")
+            + GetHelpString(9, "feeSourceAddress") +
             "\nResult:\n"
-            "{                         (json object)\n"
-            "  \"tx\" :                  (string) The serialized ProTx in hex format.\n"
-            "  \"collateralAddress\" :   (string) The collateral address.\n"
-            "  \"signMessage\" :         (string) The string message that needs to be signed with\n"
-            "                          the collateral key.\n"
+            "{                             (json object)\n"
+            "  \"tx\" :                      (string) The serialized ProTx in hex format.\n"
+            "  \"collateralAddress\" :       (string) The collateral address.\n"
+            "  \"signMessage\" :             (string) The string message that needs to be signed with\n"
+            "                              the collateral key.\n"
```

Comment:
Yep, to realign the text through the whole combined help message. Needed this because help strings had different number of whitespaces for the same param initially.

Reply:
👍